scripts/src/setUpFixture.ts

```diff
@@ -14,7 +14,9 @@ export const setUpFixture = async ({
 	logger: Logger;
 	fixture: string;
 }) => {
-	const directory = join(WORKSPACES_PATH, Math.random().toString(36).slice(2));
+	const now = new Date();
```

Comment:
We've already got a timestamp which we use throughout the script. I'd rather we just re-use that because it'll be easier to associate logs with directories: https://github.com/GregBrimble/pages-e2e-tests/blob/main/scripts/run-test.ts#L23

Reply:
yeah I know.... I really wanted something more human readable 😓

But if you strongly feel we should use that ok I can live with it 🙂 (and chance I could make that one more human readable? (without going full ISO))